CarPath/src/main/java/coordinates/CoordinateController.java

```diff
@@ -0,0 +1,97 @@
+package coordinates;
+
+import api.CarAPI;
+import javafx.scene.control.Label;
+
```

Comment:
Missing import statement `import javafx.application.Platform;` is causing a compile error and needs to be fixed.